build/template_kql/common/calculate_score.kql

```diff
@@ -15,18 +19,26 @@ extend ReliabilityScore = case(
+    Type == 'microsoft.cache/redis', (SucceededStateCount + PremiumSkuCount + EnterpriseSkuCount + AvZoneCount + NAAvZoneCount + Gt1CapacityCount) * 100 / (4 * ResourceTotal),
+    // APIM - 5 checks // score = ("Tier" + state + "capacity" + AZ + stv2platform) * 100 / (5 * ResourceTotal) 
+    Type == 'microsoft.apimanagement/service', (OtherSku + SucceededStateCount + Gt1CapacityCount + AvZoneCount + Stv2ApimCount) * 100 / (5 * ResourceTotal),
     // Storage - 2 checks // score = ( "kind" + "replication type") * 100 / (2 * ResourceTotal)    //********* Future - "access tier hot"
-    Type == 'microsoft.storage/storageaccounts' or Type == 'microsoft.classicstorage/storageaccounts', (V2StorageEnabled + RAStorageEnabled) * 100 / (2 * ResourceTotal),
+    Type == 'microsoft.storage/storageaccounts' or Type == 'microsoft.classicstorage/storageaccounts', (V2StorageEnabled + RAStorageEnabled + NARAStorageEnabled) * 100 / (2 * ResourceTotal),
     // AzureFirewall - 1 check // score = (State + AZ) * 100 / (2 * ResourceTotal)
     Type == 'microsoft.network/azurefirewalls', (AzFWSucceededStateCount + AvZoneCount + NAAvZoneCount)* 100 / (2 * ResourceTotal),
     // AFD and CDN - 1 check // score = state  * 100 / (1 * ResourceTotal)
     Type == 'microsoft.cdn/profiles' or Type == 'microsoft.network/frontdoors', (CDNEnabledStateCount + AFDEnabledStateCount) * 100 / ResourceTotal,
     // AppGW - 5 checks    // score = (state + AZ + Tier + autoscaling + capacity) * 100 / (5 * ResourceTotal)
     Type == 'microsoft.network/applicationgateways', (RunningState + AvZoneCount + NAAvZoneCount + OtherAppGwSkuCount + AutoScaleAppGwCount + Gt1CapacityCount) * 100 / (5 * ResourceTotal),
-    // LB - 2 checks  // score = (SKU + AZ) * 100 / (2 * ResourceTotal)
-    Type == 'microsoft.network/loadbalancers', (OtherSku + AvZoneCount + NAAvZoneCount) * 100 / (2 * ResourceTotal),
+    // LB - 3 checks  // score = (SKU + state + AZ) * 100 / (3 * ResourceTotal)
+    Type == 'microsoft.network/loadbalancers', (OtherSku + SucceededStateCount + AvZoneCount + NAAvZoneCount) * 100 / (3 * ResourceTotal),
+    // Public IP - 3 checks // score = (SKU + state + AZ) * 100 / (3 * ResourceTotal)
+    Type == 'microsoft.network/publicipaddresses', (OtherSku + SucceededStateCount + AvZoneCount + NAAvZoneCount) * 100 / (3 * ResourceTotal),
+    // Public IP Prefix - 3 checks // score = (SKU + state + AZ) * 100 / (3 * ResourceTotal)
+    Type == 'microsoft.network/publicipprefixes', (OtherSku + SucceededStateCount + AvZoneCount + NAAvZoneCount) * 100 / (3 * ResourceTotal),
+    // VNET Gateway - 6 checks // score = (SKU + state + capacity + vpntype + generation + active-active) * 100 / (6 * ResourceTotal)
+    Type == 'microsoft.network/virtualnetworkgateways', (AzVnetGwSkuCount + SucceededStateCount + Gt1CapacityCount + NACapacityCount + RouteVnetGwVpnTypeCount + Gen2VnetGwCount + NAGen2VnetGwCount + ActiveActiveVnetGwCount + NAActiveActiveVnetGwCount) * 100 / (6 * ResourceTotal),
```

Comment:
How about display the same new indicator related VPN Gateway on "Export" tabs?
It's helpful to review exported CSV. When review based on exported CSV, we need to understand why a score is displayed.